### PR TITLE
Migrate 'list releases' command to the new client

### PR DESCRIPTION
**Pull request by marians**
Ongoing client migration

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -406,3 +407,48 @@ func (w *WrapperV2) GetInfo(p *AuxiliaryParams) (*info.GetInfoOK, error) {
...
+		if p.ActivityName != "" {
+			params.SetXGiantSwarmActivity(&p.ActivityName)
+		}
+		if p.CommandLine != "" {
+			params.SetXGiantSwarmCmdLine(&p.CommandLine)
+		}
+		if p.RequestID != "" {
+			params.SetXRequestID(&p.RequestID)
+		}
+	}
```
Comment (kopiczko) [inline client/client.go]: That's a lot of repetition here. It looks like there can be interface extracted and then we could apply those params in a generic way.

**Review by kopiczko — APPROVED** [review]

File: client/client.go (hunk 1 above)
Comment (marians) [reply client/client.go]: +1! I really hope so.

Comment (kopiczko) [reply client/client.go]: Uncertain if we understood each other. This is in your reach. You can create this interface yourself. In Go you don't have to make struct implement interface it just has to have interface's methods.

Comment (marians) [reply client/client.go]: Yeah, I know. I just haven't taken the time to do it yet and it's something I am not used to doing.

**Pull request merged**